test/regress/regress0/model-core.smt2

```diff
@@ -1,4 +1,4 @@
-; COMMAND-LINE: --produce-model-cores
+; COMMAND-LINE: --produce-models --model-cores=simple
```

Comment:
What do you think about adding Alessandro's example as another test case for both modes (maybe in regress1)?